nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/NiFiServiceFacade.java

```diff
@@ -767,9 +767,10 @@
      *
      * @param connectionId The ID of the connection
      * @param listingRequestId The ID of the listing request
+     * @param maxResults The maximum number of flowfile summary objects to return
      * @return The ListingRequest
      */
-    ListingRequestDTO createFlowFileListingRequest(String connectionId, String listingRequestId);
```

Comment:
Have been meaning to mention this. Since this is public in an API package, I think this needs to be kept around and marked @Deprecated instead of deleted outright. Same comment on some of the other methods below.

Reply:
Yup, totally. +1  I should have thought of this, not sure why I didn't.  Good catch, thank you.  Will fix.